### ISSUE TITLE
s and m in ArcFace loss

### ISSUE THREAD

**Issue by wjtan99** [psinger/kaggle-landmark-recognition-2020-1st-place#8]
Thanks for sharing your great work. 
I noticed that you have s=45, m=0.1 by default in the loss function. In your configx.py, you have s=45, m=0.3,0.4 etc. In the original ArcFace paper, the default s=60, m=0.5.  Did you  optimize the s and m in your work?

**Comment by ChristofHenkel** [comment]
yes, s and m are important hyperparameter, which we optimized

**Comment by wjtan99** [comment]
What is the best value of your optimization? You have quite a few configx.py, so I am not sure which one is your final choice.

**Comment by ChristofHenkel** [comment]
If I remember correctly, we used slightly different m depending on if we also trained on GLDv1. Having slightly different m values per config, also helped diversity in our ensemble.

**Comment by wjtan99** [comment]
Thanks a lot for your prompt response.
